flow_action_components/GetRecordTypeInfobyObject/force-app/main/default/classes/GetRecordTypeInfobyObject.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>48.0</apiVersion>
```

Comment:
Be careful about version numbers. 48 is Spring '20. no one will be able to install this to a Winter '20 org. At this point that's not much of a problem, but generally unless there's a reason a piece of metadata has to use a prerelease api, it's safer to use the current version.

Reply:
Updated the version